### PR TITLE
[Release] Creating DLL

### PR DESCRIPTION
**Pull request by AyeshaHamad**
Creating tables using ddl.sql

### CODE REVIEW

**Comment by AyeshaHamad** [comment]
Implemented Monitoring Agent on another feature branch and then merged both to develop.

**Comment by AyeshaHamad** [comment]
Do i need to create a pull request for every branch to merge it from dev to release? or i just need to update my dev branch with latest code. and it will be automatically submit all changes even if i already made a PR.

File: linux_sql/README.md [hunk 1]
```diff
@@ -1,2 +1,336 @@
 # Linux Cluster Monitoring Agent
-This project is under development. Since this project follows the GitFlow, the final work will be merged to the master branch after Team Code Team.
+
+This project is designed to manage and administer Jarvis Linux Cluster, so that the LCA team can get hardware specifications and software utilization, memory/CPU management and generate reports according to their requirements. This project will help the LCA team to make decisions like if the system is fully utilized or not, or if they want to increase hardware/software specifications.
```
Comment (jarviscanada) [inline linux_sql/README.md:3]: well done

**Review by jarviscanada — COMMENTED** [review]
LGTM

File: linux_sql/README.md (hunk 1 above)
Comment (AyeshaHamad) [reply linux_sql/README.md:3]: thank you

**Pull request merged**